src/main/java/org/shipkit/auto/version/CommitCounter.java

```diff
@@ -44,19 +48,22 @@
      * @param gitOutput - output from "git log --pretty=oneline" command
      */
     int countCommitDelta(String gitOutput) {
+        LOG.lifecycle("countCommitDelta " + gitOutput);
         gitOutput = gitOutput.trim();
         String[] lines = gitOutput.split("\\R");
         Pattern pattern = Pattern.compile(".* Merge pull request #\\d+ from .*");
         int commits = 0;
         int mergeCommits = 0;
         for (int i = lines.length-1; i >= 0; i--) {
             String line = lines[i];
-            commits++;
-            if (pattern.matcher(line).matches()) {
-                mergeCommits++;
-                commits = 0; //reset so that we focus on merge commits
+            if(line.trim().length() != 0) {
```

Comment:
Why do we need ```if(line.trim().length() != 0)```? (trying to minimize conditional complexity)

Reply:
Because `gitOutput.split("")` returns array with one empty string. I extracted this condition to the beginning of the method.